### ISSUE TITLE
[BUG][MazPhoneNumberInput] V-model not showing correctly phone number

### ISSUE THREAD

**Issue by brunokunace** [LouisMazel/maz-ui#337]
**Describe the bug**
When try to use a value (update profile screen) with a previous value, the same not select the current country correctly

**To Reproduce**
https://codesandbox.io/s/musing-firefly-w5sgsf?file=/src/App.vue

**Expected behavior**
Show correctly and formatted number in input

**Screenshots**
Error:
<img width="867" alt="image" src="https://user-images.githubusercontent.com/16597320/186661802-8ef279b6-4eb2-4eaa-b724-6436ec0b994f.png">

Expected: 
<img width="844" alt="image" src="https://user-images.githubusercontent.com/16597320/186661903-0d7a961a-c058-47f7-bb5c-93f5d550ef70.png">

**Comment by LouisMazel** [comment]
Hi @brunokunace 

The code country detection seems have some bugs. I will check it. 
In the meantime, you have set the country code with like that:

```
<MazPhoneNumberInput v-model="phoneNumber" default-country-code="BR" />
```

**Comment by LouisMazel** [comment]
Hey @alrayyes,

This issue is solved in [v3.0.11](https://github.com/LouisMazel/maz-ui/releases/tag/v3.0.11)

Enjoy
